run

```diff
@@ -114,21 +114,31 @@ get_buildtype_str() {
 
 get_app_source_root_dir() {
   # Get source path for a given application according to HoloHub convention.
-  # Usage: ./run get_app_source_root_dir <app_name>
+  # Usage: ./run get_app_source_root_dir <app_name> [is_workflow]
```

Comment:
We should change the logic to look first for apps and then workflow directory.

Reply:
Here I assumed that a workflow is an application (end-to-end application) but just stored under different directory. I will revisit it after making sure that the workflow itself is working with HSB.

Reply:
Let's avoid updating CLIs for workflows specifically if possible. Workflows are part of a class of runnable projects including "applications" and "benchmarks" as well. Could instead:
- create an `is_workflow()` command to check a unique project name against a globbed list of workflow names
- check for workflows as a fallback like is done for benchmarks below

Impact: Adding workflow-specific args to commands is not scalable, could be confusing or tenuous for devs working on other projects
Risk: Not blocking, but could result in CLI differences if we refactor during the move to `holohub` CLI

Reply:
Tanks for the comment! I made the changes.